public/main.js

```diff
@@ -1,5 +1,40 @@
 const qrButton = document.getElementById('createqr')
 
+function createCard(data) {
+  const $qrCodeDiv = document.createElement('div')
+  const $qrCode = document.createElement('img')
+  const $name = document.createElement('h3')
+
+  $qrCode.setAttribute('src', data.qr)
+  $qrCode.setAttribute('class', 'img-thumbnail')
+  $qrCodeDiv.setAttribute('class', 'col-xs-3')
+  $qrCode.setAttribute('id', data.id)
+
+  $name.setAttribute('class', 'text-center')
+  $name.textContent = data.name
+
+  $qrCodeDiv.appendChild($qrCode)
+  $qrCodeDiv.appendChild($name)
+
+  const $cardRow = document.getElementById('cardrow')
+  $cardRow.appendChild($qrCodeDiv)
```

Comment:
all this function should do is create the card div and return it (`$qrCodeDiv`). Outside of this function, you should append to the $cardRow